sentinelone/plugin.spec.yaml

```diff
@@ -1320,6 +1438,17 @@ actions:
         example: true
         default: true
         required: true
+      operational_state:
+        title: Operational State
+        description: Agent operational state
+        type: string
+        example: na
+        enum:
+        - na
```

Comment:
Do you know what the default is when no operational_state is provided? Want to make sure we didn't change the default behavior of this action because after this change `na` will be set as the default due to it being top of the enumerated list. If passing `na` changes the output then we might want to add an `Any` or `None` item to top of list.

Reply:
Added Any as a default value for this enum.